artisatomic/__init__.py

```diff
@@ -1281,7 +1306,10 @@ def interpret_configuration(instr_orig: str) -> tuple[list[str], int, int, int,
             term_parity = (
                 term_parity + 2
             )  # this accounts for things like '3d7(4F)6d_5Pbe' in the Hillier levels. Shouldn't match these
-        instr = instr[:-1]
+            instr = instr[:-1]
```

Comment:
I think that either this indentation change, or the other instr-related change above is causing an infinite loop in the pytest checks.